lambda-notes/src/components/ListView.js

```diff
@@ -1,11 +1,27 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
+
+const NoteCardContainer = styled.div`
+  display:flex;
+  flex-wrap:wrap;
+`
+
+const NoteCard = styled.div`
+  background-color: white;
+  border: 1px solid #979797;
+  margin-right: 10px;
+  margin-top: 20px;
+  width: 190px;
+  height: 200px;
+  overflow: hidden;
+`
 
 class ListView extends Component {
   render() {
     return (
-      <div>
-        {this.props.notes.map ( note => <h2 key={Math.random()}>{note.title}</h2>) }
-      </div>
+      <NoteCardContainer>
```

Comment:
`NoteCardContainer` wrapping `NoteCard` producing a styled-component note per each note from the notes array, awesome!!!